homework/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="./main.css">
+  <link href="https://fonts.googleapis.com/css?family=Play" rel="stylesheet">
+  <title>HYF Github Respositories</title>
+</head>
+
+<body>
+  <div id="container">
+    <div id="container-left">
+      <header>
+        <section>
+          <div class=”media”>
+            <img src="hyfrepo.png" alt="HYF Repositories">
+          </div>
+        </section>
+
+        <h1>HYF Github Repositories</h1>
+        <form>
+          <select id="repo-list" name="List of HYF Repositories">
+            <option value="" hidden>Choose a Respository for details</option>
+          </select>
+        </form>
+      </header>
+    </div>
+
+    <div id="container-right">
+      <div id="up">
+        <h2>Repository's details</h2>
+        <table>
```

Comment:
You should generally only use `<table>`s for data which you actually need to display in a table (which is something you should avoid as much as possible in general, because they are rarely responsive). Just use `div`s or something.